scripts/generateDocs.mjs

```diff
@@ -2,8 +2,9 @@ import path from 'path';
 import markdownMagic from 'markdown-magic';
 import fs from 'fs';
 import https from 'https';
+import { fileURLToPath } from 'url';
 
-const __dirname = new URL('.', import.meta.url).pathname;
+const __dirname = fileURLToPath(new URL('.', import.meta.url));
```

Comment:
This file is coming from a template that all components use and individual components should not diverge.

Is there a case why this change is necessary for this issue?

Reply:
@jason-capsule42, in my case, it doesn't work on my machine.
I have something like `home/fajar/Git Project/auro-dropdown` local path. When I ran the old script, it returns `home/fajar/Git%20Project/auro-dropdown` and somehow it couldn't find the path it refer to.

<img width="1129" alt="image" src="https://github.com/AlaskaAirlines/auro-dropdown/assets/124668226/501ce4f2-af21-4cf8-82b3-6161ebbbb383">

I already asked Dale about this as well, He said it is fine to change that.

Reference: https://github.com/AlaskaAirlines/auro-radio/pull/120/commits/2fcaa20682b749e8bb7fb9340e7f1492db3c19b9

Reply:
Let't not make this change as a one off just to this component repo. @jordanjones243 @blackfalcon what do you think about this proposed change to the script. Is this something you think makes sense to change to all the generateDocs.js files in all components?